rpc/types/events.go

```diff
@@ -202,11 +219,11 @@ func fillTxAttribute(tx *ParsedTx, key []byte, value []byte) error {
 	case evmtypes.AttributeKeyEthereumTxHash:
 		tx.Hash = common.HexToHash(string(value))
 	case evmtypes.AttributeKeyTxIndex:
-		txIndex, err := strconv.ParseInt(string(value), 10, 64)
+		txIndex, err := strconv.ParseInt(string(value), 10, 32)
```

Comment:
should it parse one bit less and check the value is non-negative?

Reply:
Changed to this now:
```
strconv.ParseUint(string(value), 10, 31)
```